package.json

```
@@ -1,7 +1,6 @@
 {
   "license": "UNLICENSED",
   "devDependencies": {
-    "@getoutreach/semantic-release-circleci-orb": "^1.1.10",
     "@semantic-release/commit-analyzer": "^10.0.1",
```

suggestion: We really need to add a stencil block here.

```suggestion
    "@getoutreach/semantic-release-circleci-orb": "^1.1.10",
    "@semantic-release/commit-analyzer": "^10.0.1",
```